### PR TITLE
Add fortran language to complexity

### PR DESCRIPTION
**Pull request by noy-apiiro**
closes [LIM-24152](https://apiiro.atlassian.net/browse/LIM-24152)

Added Fortran language to this service that we can calculate complexity for it. 
Also changes in lim in [this PR ](https://github.com/apiiro/lim/pull/31144)

### CODE REVIEW

File: calculate/langauges.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ var languageToExtensions = map[Language][]string{
 	"rust":       {"rs"},
 	"scala":      {"scala", "sc"},
 	"php":        {"php", "phtml", "php3", "php4", "php5", "php7", "phps", "pht", "phar"},
+	"fortran":    {"f", "for", "f77", "f90", "f95", "f2k", "f03", "f03p", "f08", "f08p", "f15", "f20", "f18", "f2k", "f2003", "f2008", "f2015", "f2018", "fpp", "ftn", "f05", "F", "FOR", "F77", "F90", "F95", "F03", "F08", "F15", "F18", "F2K", "F2003", "F2015", "F2008", "F2018"},
```
Comment (idanbd) [inline calculate/langauges.go]: Looking at the other examples, all of them are lower case, I wonder if this should be distinct lower case values and the mechanism knows to be case insensitive (should be tested)

File: calculate/calculate_test.go [hunk 2]
```diff
@@ -69,6 +69,7 @@ func TestIncludeExcludePatterns(t *testing.T) {
 	touch(filepath.Join(basePath, "src", "svc.java"))
 	touch(filepath.Join(basePath, "src", "api.js"))
 	touch(filepath.Join(basePath, "src", "nested", "util.js"))
+	touch(filepath.Join(basePath, "src", "nested", "util2.f90"))
```
Comment (idanbd) [inline calculate/calculate_test.go:73]: Randomly writing here: I think you need to update the version in `main.go` (I'm referring to https://github.com/apiiro/code-complexity/blob/main/main.go#L14)

**Review by avig-apiiro — APPROVED** [review]
LGTM

File: calculate/calculate_test.go (hunk 2 above)
Comment (idanbd) [inline calculate/calculate_test.go:73]: Randomly writing here: lets update the README of this repository that it supports Fortran (I'm referring to https://github.com/apiiro/code-complexity/blob/main/README.md?plain=1#L105)

File: calculate/langauges.go (hunk 1 above)
Comment (noy-apiiro) [reply calculate/langauges.go]: I already checked if there is a place in the code that reference for case sensitive but I didn't find any reference for it. I'll check it again and also add a test for upper case. 
Thanks!

**Review by idanbd — APPROVED** [review]
Approving assuming you checked the case sensitive behavior

**Review by ravyolii — APPROVED** [review]

**Pull request merged**